recipes/pylint-celery/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pylint-celery" %}
+{% set version = "0.3" %}
+{% set sha256 = "41e32094e7408d15c044178ea828dd524beedbdbe6f83f712c5e35bde1de4beb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  {% set pypi_name = name.replace("_", "-") %}
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pylint >=1.0
+    - astroid >=1.0
+    - pylint-plugin-utils >=0.2.1
+
+  run:
+    - python
+    - pylint >=1.0
+    - astroid >=1.0
+    - pylint-plugin-utils >=0.2.1
+
+test:
+  imports:
+    - pylint_celery
+
+about:
+  home: https://github.com/landscapeio/pylint-celery
+  license: GPL 2
```

Comment:
Proposing the license file be added to the manifest in PR ( https://github.com/landscapeio/pylint-celery/pull/3 ). May need to resolve this in the feedstock.